spec/support/examples_group/shared_examples_for_provisioning_state_machine.rb

```diff
@@ -1,6 +1,7 @@
 shared_examples_for "common rhev state machine methods" do
   it "#customize_destination" do
     allow(@task).to receive(:get_provider_destination).and_return(nil)
+    allow(@task).to receive(:destination_image_locked?).and_return(false)
```

Comment:
This is the only part that will be problematic after the split.
You will need to extract this shared example to a shared example in the new repo